src/main/java/plugily/projects/villagedefense/arena/managers/spawner/ZombieSpawner.java

```diff
@@ -6,14 +6,21 @@
 /**
  * The interface for zombie spawner
  */
-public interface ZombieSpawner {
+public interface ZombieSpawner extends Comparable<ZombieSpawner> {
   /**
    * Get the name of the spawner
    *
    * @return the name
    */
   String getName();
 
+  /**
+   * Get the priority of the spawner (the higher the sooner)
+   */
+  default int getPriority() {
```

Comment:
Is these implementation(s) used somewhere? Or it will modifies the iteration order of `TreeSet`? If not, consider making some of related configurable features or whatever to allow priorities for zombie spawning order or anything else not just unused API availabilities that the main plugin does not uses.

Reply:
...

Reply:
Of course it will modify the iteration order of the TreeSet when adding.

Reply:
Is this needed with the default 0 priority for all spawners?

Reply:
default is 0. you can override it in a specific spawner and have it spawning first or last.
You can override it and add related configurable features. I won't add it.
It's useless in the main plugin, but useful when doing with APIs.

Reply:
I want to make an addon to add MythicMobs bosses to Village Defense
This PR is required for that to work as I want the plugin to loop through the boss spawners first

Reply:
If you think it's unused. well... you may not want the plugin to be expandable... and that is something I don't like working with.